src/ray/raylet/raylet_client.cc

```diff
@@ -0,0 +1,366 @@
+#include "raylet_client.h"
+
+#include <inttypes.h>
+#include <netdb.h>
+#include <netinet/in.h>
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/ioctl.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <sys/un.h>
+
+#include "ray/common/common_protocol.h"
+#include "ray/ray_config.h"
+#include "ray/raylet/format/node_manager_generated.h"
+#include "ray/raylet/task_spec.h"
+#include "ray/util/logging.h"
+
+using MessageType = ray::protocol::MessageType;
+
+RayletConnection::RayletConnection(const std::string &raylet_socket, int num_retries,
+                                   int64_t timeout) {
+  /* Pick the default values if the user did not specify. */
```

Comment:
`/*` -> `//`

Reply:
In other places as well.

Reply:
actually, maybe this code should just go away anyway and use `ClientConnection` instead.